docs/libcurl/libcurl-security.md

```diff
@@ -380,7 +380,7 @@ CURLOPT_TIMEOUT(3) and/or CURLOPT_LOW_SPEED_LIMIT(3) options can
 be used to mitigate against this.
 
 A malicious server could cause libcurl to download an infinite amount of data,
-potentially causing all of memory or disk to be filled. Setting the
+potentially causing all memory or disk to be filled. Setting the
```

Comment:
I would prefer to rephrase this since it's adding details where we discuss general issues. I would propose:

> potentially causing system resources to be exhausted resulting in a potential system, or application, crash.

Reply:
good point, thanks!